src/i18n/strings/en_EN.json

```diff
@@ -2114,6 +2114,7 @@
         "list_title": "Room list",
         "notification_options": "Notification options",
         "open_space_menu": "Open space menu",
+        "primary_filters": "Room list primary filters",
```

Comment:
This sounds like jargon, I don't think "primary filter" should be exposed to users

Reply:
Right, *Room list filters* is enough